src/test/java/org/entando/kubernetes/client/EntandoCoreClientTest.java

```diff
@@ -1,23 +1,28 @@
 package org.entando.kubernetes.client;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
+import com.github.tomakehurst.wiremock.client.WireMock;
 import org.entando.kubernetes.client.core.DefaultEntandoCoreClient;
 import org.entando.kubernetes.client.core.EntandoCoreClient;
+import org.entando.kubernetes.exception.web.HttpException;
+import org.entando.kubernetes.model.bundle.descriptor.*;
 import org.entando.kubernetes.model.digitalexchange.ComponentType;
 import org.entando.kubernetes.model.entandocore.EntandoCoreComponentUsage;
 import org.entando.kubernetes.utils.EntandoCoreMockServer;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
-@Tag("in-process")
-public class EntandoCoreClientTest {
+@Tag("unit")
```

Comment:
Are you sure about this?

Reply:
Yes, that class's tests were not executed. If you look at the pom you can see that in this project we execute only _unit_ and _component_ tagged tests